app/assets/stylesheets/hoge/fuga.css.scss

```diff
@@ -0,0 +1,40 @@
+.cat{}
+
+.dog {
+  color: #ff00ff;
+}
+
+p {
+  background: #ab;
+}
+
+
+span.hoge {
+        padding: 10px;
```

Comment:
__[scss_lint]__
* [Indentation] Line should be indented 2 spaces, but was indented 8 spaces

<!-- Comment by [SideCI](https://sideci.com) -->